src/plots/cartesian/layout_attributes.js

```diff
@@ -279,22 +279,22 @@ module.exports = {
         pattern: {
             valType: 'enumerated',
             // TODO could add '%H:%M:%S'
-            values: ['%w', '%H', ''],
+            values: ['day of week', 'hour', ''],
             dflt: '',
             role: 'info',
             editType: 'calc',
             description: [
                 'Determines a pattern on the time line that generates breaks.',
-                'If *%w* - Sunday-based weekday as a decimal number [0, 6].',
-                'If *%H* - hour (24-hour clock) as a decimal number [0, 23].',
-                'These are the same directive as in `tickformat`, see',
-                'https://github.com/d3/d3-time-format#locale_format',
+                'If *day of week* - Sunday-based weekday as a decimal number [0, 6].',
+                'If *hour* - hour (24-hour clock) as a decimal number [0, 23].',
+                '*day of week* and *hour* are similar to *%w* and *%H* directives',
```

Comment:
we can remove this line and the next one that points to d3-time-format

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4653/commits/c0e41044ba14f95d23bd8a758ab377cb8e6044ae.